rule-engine/rule-engine-demos-test/pom.xml

```diff
@@ -43,6 +43,12 @@
       <version>${web.tester.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.18.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
explicitly adding this version for tests to prevent weird conflicts and crashes

Reply:
wasn't the version leaked through another dependency? then we should rather exclude it there.

Reply:
much better 🥛